implementation/src/test/java/test/io/smallrye/openapi/runtime/scanner/entities/EnumRequiredContainer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package test.io.smallrye.openapi.runtime.scanner.entities;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+/**
+ * @author Marc Savy {@literal <marc@rhymewithgravy.com>}
```

Comment:
Nice of you to give away the credit.... or is it the blame?? ;)

Reply:
:-) I noticed this after I already pushed the branch.